server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala

```diff
@@ -16,7 +16,7 @@ case class PlayServerOptions(
     playBodyParsers: PlayBodyParsers
 ) {
 
-  lazy val logger: Logger = Logger(getClass.getPackageName)
+  lazy val logger: Logger = Logger("sttp.tapir.server.play")
```

Comment:
How about `sttp.tapir.server.play.getClass` ?

Reply:
Ok, I see, it doesn't work as well :(

Reply:
Maybe `this.getClass.getPackage.getName` ?

Reply:
@ghostbuster91 updated